BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("@gapic_generator_python_pip_deps//:requirements.bzl", "requirement")
+
+py_binary(
+    name = "gapic_generator_python",
+    srcs = glob(["gapic/**/*.py"]),
+    data = glob(["gapic/**/*.j2"]),
+    main = "gapic/cli/generate.py",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@com_google_protobuf//:protobuf_python",
+        requirement("click"),
+        requirement("google-api-core"),
+        requirement("googleapis-common-protos"),
+        requirement("grpcio"),
+        requirement("protobuf"),
+        requirement("jinja2"),
+        requirement("pypandoc"),
+        requirement("PyYAML"),
+    ],
+    python_version = "PY3",
```

Comment:
Is there any way we can be more specific about the python version requirement? Some of the code in the generator requires features from >=3.6

Reply:
There are ways to be more specific, but they have their own issues (i.e. make the build less hermetic and less portable). Practically speaking, it should be picking the latest installed version of python on the host machine, and if there is 3.6+ bazel will pick it up.

Reply:
Okay. Good to know.